src/core.c

```diff
@@ -1383,6 +1383,8 @@ void EndDrawing(void)
     }
 #endif
 
+    rlglCheckErrors();
```

Comment:
I prefer not forcing the errors check for the cost it can suppose, I'd just leave usage up to the user.